vagrant/plugins/vagrant-hypconfigmgmt/lib/vagrant-hypconfigmgmt/command.rb

```diff
@@ -282,6 +284,13 @@ def ensure_vagrant_box_type_configured(env)
       update_settings(settings)
     end
 
+
+    def ensure_default_domain_configured(env)
+      settings = retrieve_settings()
+      settings['hostmanager']['default_domain'] ||= DEFAULT_DOMAIN
```

Comment:
> Is this all for resetting the tests?
  
no this is to ensure that if the settings don't have this item the default one is to be written to the local.yml configuration file